client/src/js/services/VoucherService.js

```diff
@@ -26,6 +26,7 @@ function VoucherService(Api) {
     { id: 8, text: 'VOUCHERS.SIMPLE.PURCHASES', incomeExpense: 'expense', prefix: 'ACHAT' }
   ];
 
+  service.url = '/vouchers/';
```

Comment:
You shouldn't need this line... [Line 14 of this file](https://github.com/mbayopanda/bhima-2.X/blob/b029bd576b38d8b0209b60f235d5b9f63e08abfc/client/src/js/services/VoucherService.js#L14) should attach the `/vouchers/` URL automatically.